src/main/webapp/app/quiz/edit/drag-and-drop-question/edit-drag-and-drop-question.component.html

```diff
@@ -53,6 +53,30 @@
                     <option value="ALL_OR_NOTHING">{{'arTeMiSApp.quizExercise.scoringType.all_or_nothing' | translate}}</option>
                     <option value="PROPORTIONAL_WITH_PENALTY">{{'arTeMiSApp.quizExercise.scoringType.proportional_with_penalty' | translate}}</option>
                 </select>
+
+                <ng-template #info let-c="close" let-d="dismiss">
```

Comment:
Am I correct, that the two modals are identical?
What about moving it to its own component instead?

What about creating a `<jhi-quit-scoring-info-dialog>` Component that can be used in Both the Edit Drag and Drop Question Component and the Edit Multiple Choice Question Component.

This Angular Component would display the button and trigger the modal. Reducing the dependencies on `NgbModal` and keeping the Changes to the other components to one line in the HTML Files.